GPU/Directx9/ShaderManagerDX9.cpp

```diff
@@ -342,7 +342,7 @@ void ShaderManagerDX9::VSUpdateUniforms(u64 dirtyUniforms) {
 			flippedMatrix[12] = -flippedMatrix[12];
 		}
 
-		ConvertProjMatrixToD3D(flippedMatrix, invertedX, invertedY);
+		ConvertProjMatrixToD3D(flippedMatrix, invertedX, !invertedY);
```

Comment:
Hm.  If this is correct, I'd rather reverse the check inside the function rather than invert this value.    This is the only call, and the argument inside and variable outside are both named `invertedY`.

I dislike the idea of them having opposite meanings - we should either inverse the check inside the func, or change one of the names, or reverse the way the variable is initially set here.

-[Unknown]

Reply:
Thinking about this more, is it a problem that this invert is in non-GL specific code?

https://github.com/hrydgard/ppsspp/blob/7790e61b12d016e5d9b97895f9d51f605398ff69/GPU/Common/SoftwareTransformCommon.cpp#L524-L531

(though I thought we used top down, or basically upside down, in GL buffers except the backbuffer...)

-[Unknown]

Reply:
Yeah, that looks really broken...